report_builder/api/views.py

```diff
@@ -147,10 +147,10 @@ def post(self, request):
                         'field_verbose': field,
                         'field_type': 'Property',
                         'field_choices': None,
-                        'can_filter': True if filters is None or new_field.name in filters else False,
+                        'can_filter': True if filters is None or field in filters else False,
```

Comment:
@grantjenks I'm having a hard time trying to think of a test for this. Because `new_fields` did not exist it would have given a referenced before initialized error, but I'm not sure how to test for it here.. Do you have any ideas?